src/go/rpk/pkg/cli/cmd/debug/bundle/bundle.go

```diff
@@ -201,6 +216,44 @@ func NewCommand(fs afero.Fs) *cobra.Command {
 	return command
 }
 
+// uploadBundle will send the file located in 'filepath' by issuing a PUT
+// request to the 'uploadURL'.
+func uploadBundle(ctx context.Context, filepath, uploadURL string) error {
+	uploadFile, err := os.Open(filepath)
+	if err != nil {
+		return fmt.Errorf("unable to open the file %q: %v", filepath, err)
+	}
+	defer uploadFile.Close()
+
+	// Stat only to get content length.
+	stat, err := uploadFile.Stat()
+	if err != nil {
+		return fmt.Errorf("unable to get the stat the file %q: %v", filepath, err)
+	}
+
+	cl := httpapi.NewClient(
+		httpapi.Err4xx(func(code int) error { return &S3EndpointError{HTTPCode: code} }),
+		httpapi.Headers(
+			"Content-Length", fmt.Sprintf("%v", stat.Size()),
```

Comment:
This is unnecessary, Go does this automatically

Reply:
I had no idea, thanks!